src/test/java/no/nav/tag/sykefravarsstatistikk/api/provisjonering/DataverehusRepositoryJdbcTest.java

```diff
@@ -40,6 +43,30 @@ public void tearDown() {
         cleanUpTestDb(namedParameterJdbcTemplate);
     }
 
+    @Test
+    public void hentSykefraværsstatistikkLand_lager_sum_og_returnerer_antall_tapte_og_mulige_dagsverk() {
+        insertSFStatLandInDvhTabell(namedParameterJdbcTemplate, 2018, 4, 5, 100);
+        insertSFStatLandInDvhTabell(namedParameterJdbcTemplate, 2018, 4, 10, 100);
+        insertSFStatLandInDvhTabell(namedParameterJdbcTemplate, 2019, 1, 1, 10);
+
+        List<SykefraværsstatistikkLand> sykefraværsstatistikkLand =
+                repository.hentSykefraværsstatistikkLand(new ÅrstallOgKvartal(2018, 4));
+
+        assertTrue(sykefraværsstatistikkLand.size() == 1);
```

Comment:
Veldig pirkete, _men_ kunne vi ha brukt assertj sin assert i stedet? Da blir det sånn:
```
assertThat(sykefraværsstatistikkLand.size()).isEqualTo(1);
```
eller enda bedre
```
assertThat(sykefraværsstatistikkLand).hasSize(1);
```
Vi hadde en diskusjon på det i vår, og konkluderte med at assertj var litt tydeligere. I dette tilfellet gjør det ikke så mye, men når man sammenligner to objekter blir det tydeligere hvilket objekt som er "fasiten" og hvilket objekt vi vil sjekke. Greit å være konsekvent :)

Reply:
👍